docs/excel/excel-add-ins-worksheets.md

```diff
@@ -295,6 +295,56 @@ function onWorksheetChanged(eventArgs) {
 }
 ```
 
+## Sorting events (preview)
```

Comment:
```suggestion
## Handle sort-related events (preview)
```

Reply:
@kbrandl, How do you feel about calling this "Handle sorting events"? The "related" part makes the events seem distant to the actual sorting.

Reply:
That sounds good to me! Be sure to update the hyperlink (text and URL) in the other article that points to this section.